src/reducers/views/i18n.js

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2016 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {combineReducers} from 'redux';
+
+import Config from 'config/index';
+import {UsersTypes} from 'constants';
+
+function locale(state = Config.DefaultLocale, action) {
+    switch (action.type) {
+    case UsersTypes.RECEIVED_ME:
+        return action.data.locale;
```

Comment:
Does the locale need to be stored separately here? We can just get it from the current user if that exists